ql/src/test/org/apache/hadoop/hive/ql/lockmgr/TestDbTxnManager2.java

```diff
@@ -3754,7 +3754,7 @@ private void testRenameTable(boolean blocking) throws Exception {
     }
     driver2.lockAndRespond();
     locks = getLocks();
-    Assert.assertEquals("Unexpected lock count", 1, locks.size());
+    Assert.assertEquals("Unexpected lock count", 2, locks.size());
```

Comment:
what's the other lock we are expecting here, should we check this in a test?

Reply:
one lock on the source table and another one on the destination.

Reply:
please add the corresponding check in a test

Reply:
ack